src/app-retrospective.c

```diff
@@ -180,6 +181,16 @@ i7_app_get_inform_command_line(I7App *self, const char *version_id, int format,
 		g_ptr_array_add(builder, g_strdup("-internal"));
 	g_ptr_array_add(builder, internal_path);
 
+	GSettings *prefs = i7_app_get_prefs(self);
+	g_autofree char *added_nest_dir = g_settings_get_string(prefs, PREFS_ADDED_NEST);
+	if (added_nest_dir) {
```

Comment:
Note this probably doesn't do what you expected — it checks whether `added_nest_dir` is null, not whether it is the empty string. So when the setting isn't present, we're adding a `-nest` without any path, which the compiler doesn't like. (No worries, I'll fix this up before merging.)